CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -468,17 +471,6 @@ public static function getPartialPaymentWithType($entityId, $entityName = 'parti
     if ($contributionId && $financialTypeId) {
 
       $value = CRM_Contribute_BAO_Contribution::getContributionBalance($contributionId, $lineItemTotal);
-
-      $paymentVal = $value;
-      if ($returnType) {
-        $value = array();
-        if ($paymentVal < 0) {
-          $value['refund_due'] = $paymentVal;
-        }
-        elseif ($paymentVal > 0) {
-          $value['amount_owed'] = $paymentVal;
-        }
-      }
```

Comment:
@eileenmcnaughton umm the 'Record Payment' b/o form still uses these amounts: 
```
CRM//Contribute/Form/AdditionalPayment.php:112:    if (!empty($paymentInfo['refund_due'])) {
CRM//Contribute/Form/AdditionalPayment.php:113:      $paymentAmt = $this->_refund = $paymentInfo['refund_due'];
CRM//Contribute/Form/AdditionalPayment.php:609:      if (!empty($paymentInfo['refund_due'])) {
CRM//Contribute/Form/AdditionalPayment.php:610:        $this->_refund = $paymentInfo['refund_due'];
```
to decide if its refund or owed amount.

Won't it cause a regression?

Reply:
@monishdeb they are calling a different fn

```
    $paymentDetails = CRM_Contribute_BAO_Contribution::getPaymentInfo($this->_id, $this->_component, FALSE, TRUE);
```

Reply:
an no I see - it should already be broken since they don't pass 'TRUE' & the default is false

```
    $paymentInfo = CRM_Core_BAO_FinancialTrxn::getPartialPaymentWithType($this->_id, $entityType);
```

Reply:
Actually the default is TRUE if you look in this line https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/BAO/FinancialTrxn.php#L451

Reply:
Ug you are right - well I'll drop out that commit then

Reply:
@monishdeb OK I dropped the param but assume true not false now